live-editing/configs/GridConfigGenerator.ts

```diff
@@ -1445,6 +1446,28 @@ export class GridConfigGenerator implements IConfigGenerator {
             })
         }));
 
+        configs.push(new Config({
+            component: 'GridWithRatingComponent',
+            additionalFiles: ['/src/app/directives/prevent-scroll.directive.ts', '/src/app/data/nwindData.ts'],
+            additionalDependencies: ['igniteui-webcomponents'],
+            appModuleConfig: new AppModuleConfig({
+                imports: ['IgxGridModule', 'IgxToastModule', 'GridWithRatingComponent', 'IgxPreventDocumentScrollModule'],
+                ngDeclarations: ['GridWithRatingComponent'],
+                ngImports: ['IgxPreventDocumentScrollModule', 'IgxGridModule', 'IgxToastModule']
+            })
+        }));
+
+        configs.push(new Config({
+            component: 'RatingInFormComponent',
+            additionalFiles: [],
+            additionalDependencies: ['igniteui-webcomponents'],
+            appModuleConfig: new AppModuleConfig({
+                imports: ['IgxButtonModule', 'RatingInFormComponent', 'IgxCardModule'],
```

Comment:
This also needs 'IgcFormsModule' to be added to the modules, however, when I try to add it it says:

 Error: Error: IgcFormsModule path is missing for GridWithRatingComponent config!
    at C:\Users\SStoychev\Documents\Git\igniteui-angular-samples\node_modules\igniteui-live-editing\LiveEditingManager.js:65:33

The module is part of igniteui-angular

Reply:
Resolved